spec/factories/state_file_md_intakes.rb

```diff
@@ -129,10 +129,9 @@
     primary_first_name { "Mary" }
     primary_middle_initial { "A" }
     primary_last_name { "Lando" }
-    primary_birth_date { Date.new(1950, 0o1, 0o1) } # matches the bday in md_minimal.json
+    primary_birth_date { Date.new(1950, 01, 01) } # matches the bday in md_minimal.json
     primary_signature_pin { '12345' }
-    primary_esigned_at { DateTime.new(2024, 12, 19, 12)}
-    primary_esigned { 'yes' }
+    primary_esigned_at { DateTime.now }
```

Comment:
i'll probably just remove this & add this to the spec that needed it

Reply:
That sounds good